tools/install-powershell.ps1

```diff
@@ -79,6 +79,68 @@ Function Remove-Destination([string] $Destination) {
     }
 }
 
+Function IsPathInRegistryPath([string] $Path) {
+    #Remove ending DirectorySeparatorChar for comparison purposes
+    $Path = [System.Environment]::ExpandEnvironmentVariables([regex]::Replace($Path, '\\;$|\\$', ''));
+    $InstalledPaths = @()
+    #Possible gotcha: don't use Get-ItemPropertyValue (see https://github.com/PowerShell/PowerShell/issues/5906)
+    #Possible gotcha: coerce the (possibly null) property to string before calling ExpandEnvironmentVariables
+    [string] $HKCURegKeyPath = [System.Environment]::GetEnvironmentVariable('PATH','User')
+    $InstalledPaths += @([System.Environment]::ExpandEnvironmentVariables(($HKCURegKeyPath)).Split(';'))
+    [string] $HKLMRegKeyPath = [System.Environment]::GetEnvironmentVariable('Path','Machine')
+    $InstalledPaths += @([System.Environment]::ExpandEnvironmentVariables(($HKLMRegKeyPath)).Split(';'))
+    #Remove ending DirectorySeparatorChar in all items of array for comparison purposes
+    $InstalledPaths = $InstalledPaths.ForEach( { $_.Replace('\\$', '') })
+    if ($InstalledPaths -icontains $Path) {
+        return $True
+    }
+    return $False
+}
+
+Function SavePathInRegistryKey([Microsoft.Win32.RegistryKey] $Key, [string] $Path) {
```

Comment:
```suggestion
Function Set-PathInRegistryKey([Microsoft.Win32.RegistryKey] $Key, [string] $Path) {
```

Reply:
You don't have to reveal you implementation in the command name.

Reply:
And use the PowerShell naming even for functions which are not exposed

Reply:
You are a much better man than I.  I tend to avoid using a "-" in any function that wont be public to avoid the discoverability linting message.

Reply:
Your suggestion on using aprooved verb for all functions, public or not, was implemented in the latest psuh.  I chose to go with Add-Path (~proposed), Test-PathInRegistryPath (===proposed) and Update-Path (===proposed).